source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_System_Settings_Explained.adoc

```diff
@@ -47,14 +47,13 @@ The following table details the options available on the *System* tab of the *Ne
 | If OS supports hotplugging, no. Otherwise, yes.
 
 
-|*BIOS Type*
-a|Specifies the chipset and BIOS type. Defaults to the cluster's default BIOS type. Options are:
+|*Chipset/Firmware Type*
+a|Specifies the chipset and firmware type. Defaults to the cluster's default chipset and firmware type. Options are:
 
-* *Cluster default* The BIOS type defined under General cluster settings.
+* *I440FX Chipset with BIOS* Legacy BIOS.
 * *Q35 Chipset with Legacy BIOS* Legacy BIOS without UEFI (Default for clusters with compatibility version 4.4)
 * *Q35 Chipset with UEFI BIOS* BIOS with UEFI
 * *Q35 Chipset with SecureBoot* UEFI with SecureBoot, which authenticates the digital signatures of the boot loader
-* *Legacy* i440fx chipset with legacy BIOS
```

Comment:
the changes here look good, how about fixing `Q35 Chipset with Legacy BIOS` and `Q35 Chipset with SecureBoot` as mentioned above?